src/utils/thread_helper.cpp

```diff
@@ -920,6 +920,43 @@ void ThreadHelper::DetachGracefullyCallback(void)
 }
 
 #if OTBR_ENABLE_TELEMETRY_DATA_API
+#if OTBR_ENABLE_BORDER_ROUTING
+void ThreadHelper::GetNodeDivergenceInfo(threadnetwork::TelemetryData_NodeDivergenceInfo *aNodeDivergenceInfo)
```

Comment:
I think the netdata and rloc16 are already included in telemetry data. Do we want to process them here again?

Reply:
In some system, e.g. Android, the network data is not able to be uploaded, we might need to explicitly get this metrics here.